sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -619,6 +621,42 @@ public void testSinkWithSendErrors() throws Throwable {
     }
   }
 
+  @Test
+  public void testSourceDisplayData() {
+    KafkaIO.Read<Integer, Long> read = mkKafkaReadTransform(10, null);
+
+    DisplayData displayData = DisplayData.from(read);
+
+    assertThat(displayData, hasDisplayItem("topics", "topic_a,topic_b"));
+    assertThat(displayData, hasDisplayItem("enable.auto.commit", false));
+    assertThat(displayData, hasDisplayItem("bootstrap.servers", "myServer1:9092,myServer2:9092"));
+    assertThat(displayData, hasDisplayItem("value.deserializer",
+        org.apache.kafka.common.serialization.ByteArrayDeserializer.class.getName()));
+    assertThat(displayData, hasDisplayItem("key.deserializer",
```

Comment:
oh, yeah, we should drop these two serializers too. They are are implementation details.